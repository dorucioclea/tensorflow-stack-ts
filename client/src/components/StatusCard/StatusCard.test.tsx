// Jest snapshot test
import * as React from "react";
import * as TestRenderer from "react-test-renderer";
import { StatusCard } from ".";

test("jest works", () => {
    expect(1).toBeTruthy();
});

describe("StatusCard testing", () => {

    test("StatusCard snapshot is ok", () => {
        const component = TestRenderer.create(
            <StatusCard />,
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

