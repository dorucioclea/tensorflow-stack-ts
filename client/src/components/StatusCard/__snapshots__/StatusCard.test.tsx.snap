// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusCard testing StatusCard snapshot is ok 1`] = `
<div
  className="ant-card ant-card-bordered"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Server Status
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <span>
      Loading data...
    </span>
  </div>
</div>
`;
